### PR TITLE
Work on #57 - Start of scheduling notebook example

### PR DESCRIPTION
**Pull request by galleon**
First basic implementation for discussion

### CODE REVIEW

**Comment by dbarbier** [comment]
Can you please provide a binder link to your notebook?

**Comment by galleon** [comment]
[Binder](https://mybinder.org/v2/gh/airbus/scikit-decide/binder?urlpath=git-pull%3Frepo%3Dhttps%253A%252F%252Fgithub.com%252Fgalleon%252Fscikit-decide%26urlpath%3Dtree%252Fscikit-decide%252Fnotebooks%252Fscheduling_demo.ipynb%26branch%3Dscheduling)

File: notebooks/scheduling_demo.ipynb [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+    "\n",
+    "Task 0 has a duration of 1 hour\n",
+    "\n",
+    "Task 1 depends on 0 and has a duration of 3 hours\n",
+    "\n",
+    "Task 2 depends on 0 and has a duration of 1 hours\n",
+    "\n",
+    "Task 3 depends on 2 and has a duration of 3 hours\n",
+    "\n",
+    "Task 4 depends on 1 & 3 and has a duration of 2 hours"
```
Comment (dbarbier) [inline notebooks/scheduling_demo.ipynb:43]: Tasks are numbered from 0 to 4 here but from 1 to 5 in code below.

**Comment by dbarbier** [comment]
Why has this PR been closed?

**Comment by galleon** [comment]
Renamed and PR recreated as #66

**Comment by dbarbier** [comment]
PRs can be renamed, there is an Edit button at the top.

**Pull request closed without merging**